src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCF.java

```diff
@@ -0,0 +1,759 @@
+                classifiedVariants.stream()
+                        .collect(Collectors.groupingBy(EvalVariant::getAssignedConcordanceState,
+                                () -> new EnumMap<>(ConcordanceState.class),
+                                Collectors.groupingBy(classifiedEvalVariant ->
+                                                StructuralVariantType.valueOf(classifiedEvalVariant.getVariant().getAttributeAsString(VCFConstants.SVTYPE, "")),
+                                        () -> new EnumMap<>(StructuralVariantType.class),
+                                        Collectors.counting())));
+        summaryByConcordanceThenBySvType.forEach((concordanceState, countsBytype) -> {
+            final StringBuilder concordanceStateAccountingLine = new StringBuilder(String.format("%-25s:  ", concordanceState.name()));
+            final EnumMap<StructuralVariantType, Long> zeroInitializedEnumMap = SVUtils.getZeroInitializedEnumMap(StructuralVariantType.class);
+            countsBytype.forEach(zeroInitializedEnumMap::put);
+            zeroInitializedEnumMap.forEach((k, v) -> concordanceStateAccountingLine.append( String.format("%10d", v)) );
+            logger.info(concordanceStateAccountingLine.toString());
+        });
+
+        // for FN values (motivation: using truth set from PacBio SV calls, the FN number is going to be high anyway for call sets based off short reads)
+        final EnumMap<StructuralVariantType, Long> zeroInitializedEnumMap = SVUtils.getZeroInitializedEnumMap(StructuralVariantType.class);
+        final StringBuilder falseNegativeLine = new StringBuilder(String.format("%-25s:  ", ConcordanceState.FALSE_NEGATIVE.name()));
+        Utils.stream(truthTree.iterator())
+                .flatMap(e -> e.getValue().stream())
+                .filter(classifiedTruth -> classifiedTruth.getAttributeAsString(TEMPORARY_CONCORDANCE_STRING_FOR_TRUTH, "").equals(ConcordanceState.FALSE_NEGATIVE.getAbbreviation()))
+                .collect(Collectors.groupingBy(truth -> StructuralVariantType.valueOf(truth.getAttributeAsString(VCFConstants.SVTYPE, "")),
+                        () -> new EnumMap<>(StructuralVariantType.class), Collectors.counting()))
+                .forEach(zeroInitializedEnumMap::put);
+        zeroInitializedEnumMap.forEach((k,v) -> falseNegativeLine.append( String.format("%10d", v)) );
+
+        logger.info(falseNegativeLine.toString());
+    }
+
+    /**
+     * BED file contains only: TP, FP, FILTERED_TRUE_NEGATIVE (record correctly filtered), and FILTERED_FALSE_NEGATIVE.
+     *
+     * False negatives are purposefully ignored because when first implemented, the truth set is from PacBio calls,
+     * with a sensitivity that short read technology has no way of catching up to, generally speaking.
+     */
+    private void writeConcordanceToBedFile() {
+        try (final OutputStreamWriter writer = new OutputStreamWriter(new BufferedOutputStream(
```

Comment:
To me makes more sense

```
try (final BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(BucketUtils.createFile(outBed)))) {
...
    writer.println(lineStr);
    // rather than writer.write(lineString + "\n");
}
```
In any case whould you need a ```BufferedOutputStream```?.

Reply:
done

Reply:
Ah yeah, I assumed that BufferedReader has a readLine BufferedWriter would have some "writeLine". I guess then you need to do something like:

```
writer.write(lineStr)
writer.newLine();
```

or perhaps use ```PrintWriter``` instead.